pkg/synopsysctl/utils.go

```diff
@@ -236,12 +236,13 @@ func getInstanceInfo(mock bool, name string, crdType string, crdName string, inp
 
 	// Check Number of Arguments
 	if crdScope != apiextensions.ClusterScoped && len(namespace) == 0 {
-		return "", "", "", fmt.Errorf("namespace to create an '%s' instance needs to be provided", inputNamespace)
+		return "", "", "", fmt.Errorf("namespace to create/update/delete an '%s' instance needs to be provided", inputNamespace)
```

Comment:
namespace for instance '%s" needs to be provided.

Reply:
done